### PR TITLE
Update for ActivitySlip

### PR DESCRIPTION
**Pull request by alexdia25**

### CODE REVIEW

**Comment by alexdia25** [comment]
@jaspermondie

File: MYOB.API.SDK/SDK/Contracts/Version2/Contact/Customer.cs [hunk 1]
```diff
@@ -16,10 +14,5 @@ public class Customer : Contact
         /// The paying details applied to the Customer
         /// </summary>
         public CustomerPaymentDetails PaymentDetails { get; set; }
-
-        /// <summary>
-        /// The Foreign Currency associated with the entity
-        /// </summary>
-        public CurrencyLink ForeignCurrency { get; set; }
```
[the remark this answers is not in the document]
Comment (chips5k) [reply MYOB.API.SDK/SDK/Contracts/Version2/Contact/Customer.cs]: nope!  - i'll revert it

**Comment by alexdia25** [comment]
@jaspermondie changes must have been pulled in unintentionally. Have reverted 😄

File: appveyor.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: 2018.1.{build}
+version: 2018.3.{build}
```
Comment (jaspermondie) [inline appveyor.yml]: The SDK release versions do not tie in to the ARL release versions

**Comment by chips5k** [comment]
Not sure what you mean - can you elaborate?

File: appveyor.yml (hunk 2 above)
Comment (alexdia25) [reply appveyor.yml]: fixed

**Pull request merged**